src/seedu/addressbook/commands/ViewFeesCommand.java

```diff
@@ -46,16 +46,6 @@ public CommandResult execute() {
         }
     }
 
-    @Override
```

Comment:
Sorry to bother, but code belongs to code, documentation belongs to documentation. If you want to push code, create a new PR. This is so that we can properly account for changes in the code base in the master branch

Reply:
Oops sorry! Noted

Reply:
Er, revert this change then I'll merge it through